integration_tests/custom_reporters/__tests__/add-fail-test.js

```diff
@@ -0,0 +1,9 @@
+const add = require('../add');
+
+describe('CustomReporters', () => {
+    test('adds fail', () => {
```

Comment:
The pattern for `'integration_tests/*/**/'` isn't in `scripts/prettier.js`. I'll make a PR for this :+1: